mr/src/main/java/org/apache/iceberg/mr/mapreduce/IcebergInputFormat.java

```diff
@@ -165,13 +166,15 @@ private static void checkResiduals(CombinedScanTask task) {
     private Iterator<FileScanTask> tasks;
     private T currentRow;
     private CloseableIterator<T> currentIterator;
+    private FileIO fileIO;
 
     @Override
     public void initialize(InputSplit split, TaskAttemptContext newContext) {
       Configuration conf = newContext.getConfiguration();
       // For now IcebergInputFormat does its own split planning and does not accept FileSplit instances
       CombinedScanTask task = ((IcebergSplit) split).task();
       this.context = newContext;
+      this.fileIO = new HadoopFileIO(this.context.getConfiguration());
```

Comment:
Nit: we use `this.` to distinguish instance fields when setting them, but there's no need when accessing them.

Reply:
thanks, I'll keep that in mind :)

Reply:
Good to know! Do we document these style preferences anywhere (using `this`, when we use the term `project` vs `select`, etc)? @rdblue